mmv1/templates/terraform/examples/certificate_manager_self_managed_certificate.tf.erb

```diff
@@ -0,0 +1,42 @@
+resource "google_certificate_manager_certificate" "<%= ctx[:primary_resource_id] %>" {
+  name        = "<%= ctx[:vars]['cert_name'] %>"
+  description = "The default cert"
+  scope       = "EDGE_CACHE"
+  self_managed {
+    pemCertificate = "-----BEGIN CERTIFICATE-----
```

Comment:
Are we able to re-use anything in our [test-fixtures](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/utils/test-fixtures) directory? Or would you want to maybe make files there for these?

>         Error: Invalid multi-line string
> 
>         Quoted strings may not be split over multiple lines. To produce a multi-line
>         string, either use the \n escape to represent a newline character or use the
>         "heredoc" multi-line template syntax.

Reply:
Created `private-key.pem` and `cert.pem` in a `test-fixtures/certificatemanager` directory.